std/root.ln

```diff
@@ -2583,26 +2583,84 @@ export fn prune(n: Node<any>): Tree<any> {
   return n.tree;
 }
 
+// Takes the given node and copies that node and all of its children into a new tree with the
+// current node as the root. Unfortunately a sequential algorithm right now.
+export fn toSubtree(n: Node<any>): Tree<any> {
+  const val = n.tree.vals[n.id].getR();
+  let outTree = newTree(val);
+  let children = n.tree.children[n.id].getOr(new Array<int64> []);
+  if children.length().gt(0) {
+    let parentIds = [0].repeat(children.length());
+    //seqdo(newseq(pow(2, 64).getOr(0)), fn (): bool {
```

Comment:
Oh, I was debugging an infinite loop at one point, but I should restore this instead of limiting subtrees to 100 nodes.